### DISCUSSION TITLE
PAID WORK - For Freelancer - Enhance GrapesJS Image Uploader

### DISCUSSION THREAD

**Discussion by Alababdiy** [GrapesJS/grapesjs#5628] [General]
We need expret who can Integrate Cropper.js for image cropping functionality. Implement AWS S3 storage for image uploads.
check SoW doc:
https://docs.google.com/document/d/1H3Q7qlqYZAvmwb-Uqsy2nG2fYZxG7L8JVNMBEo4Tuas/edit

**Comment by mmm1k4** [comment]
Hi!

I am interested in the job. I already did similar task with grapesjs in
commercial app. Integration of the images and data files upload and also
custom extension to improve or change the image using React and vanilla js.

I am a freelance software developer and open for business and with the
common prices. I can show you my portfolio of projects at the view call

I would be happy to discuss furter the details
Please reply to this e-mail, or catch me at the working e-mail:
***@***.***

Best reagards
Michaela


čt 18. 1. 2024 v 12:07 odesílatel Muli-Users ***@***.***>
napsal:
